cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,156 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+
+.. :section-numbering: true
+
+.. _admin-installation-azure-hdinsight:
+
+=========================================
+Installation on Microsoft Azure HDInsight
+=========================================
+
+Introduction
+============
+
+This section describes installing CDAP on `Microsoft Azure HDInsight
+<https://azure.microsoft.com/en-us/services/hdinsight/>`__ using
+Microsoft's website to:
+
+- Create an HBase cluster;
+- Install CDAP as an application on the cluster; and then
+- Verify that CDAP is installed and running by using the CDAP UI.
+
+Information on Microsoft Azure HDInsight is `available online
+<https://docs.microsoft.com/en-us/azure/hdinsight/>`__.
+
+CDAP |short-version| is compatible with Microsoft Azure HDInsight.
+
+
+Creating the Cluster
+====================
+
+1. Log onto Microsoft Azure and navigate to *HDInsight clusters* (under *Intelligence + analytics*):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-1.png
+      :figwidth: 100%
+      :width: 611px
+      :class: bordered-image
+
+      **Microsoft Azure:** Finding HDInsight
+
+#. Create an *HDInsight cluster* by clicking "Create". Enter a *Cluster name* (such as
+   *cdap-cluster*). Under *Cluster type*, select "HBase". (To continue, click *Select* to
+   close the *Cluster configuration* window):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster
+
+#. Enter the required basic settings: *Cluster login password*, *Resource group*, and *Location*:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.png
+      :figwidth: 100%
+      :width: 674px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Configuring basic settings
```

Comment:
I think we need to add a step here about cluster size.  As noted in our description, at least the default of 4 D3 v2 Region nodes are required.  We should recommend that this is enough Yarn Capacity to run very simple examples, and they will need more for anything serious.